### DISCUSSION TITLE
Is it possible to load babel config file with a custom name?

### DISCUSSION THREAD

**Discussion by swallowtail62** [storybookjs/storybook#17070] [Help]
I currently use Next.js v12 and swc as compiler and Storybook v6.4.
To use swc, can't put babel config file (e.g. `.babelrc`, `.babelrc.json`, ...). Because if Next.js detects those files, then stop using swc.
https://nextjs.org/docs/messages/swc-disabled

I want to use Storybook babel v7 mode, but can I make Storybook load babel config files (e.g. `.babelrc.storybook`) in the project root that next.js doesn't load by default?
https://storybook.js.org/docs/react/configure/babel#v7-mode

**Comment by allanlasser** [comment]
I solved this in my Next.js project by moving my Babel config file into the `.storybook` directory. 

Following [Storybook’s docs](https://storybook.js.org/docs/react/configure/babel#custom-config-file) :
> You can also place a .storybook/.babelrc file to use a unique configuration for Storybook only.
